### PR TITLE
Get total vote count helper function

### PR DESCRIPTION
**Pull request by sreyaguha1**

### CODE REVIEW

File: policykit/policyengine/models.py [hunk 1]
```diff
@@ -57,6 +57,11 @@ class Community(PolymorphicModel):
     def notify_action(self, action, policy, users):
         pass
 
+        #   def get_users(self, users):
```
Comment (amyxzhang) [inline policykit/policyengine/models.py:60]: Can you remove this commented out code from the PR?

File: policykit/policyengine/models.py [hunk 2]
```diff
@@ -229,6 +234,22 @@ def get_number_votes(self, value=0, users=None):
         else:
             votes = NumberVote.objects.filter(number_value=value, proposal=self)
         return votes
+    
+    def get_total_vote_count(self, vote_type, vote_number = 1, users = None):
```
Comment (amyxzhang) [inline policykit/policyengine/models.py:238]: I don't think this should be on the datastore object. shouldn't it be on the proposal object?

Comment (sreyaguha1) [reply policykit/policyengine/models.py:238]: oops yes sorry!! this was a mistake

**Comment by amyxzhang** [comment]
Hi @sreyaguha1 can you fix the issues and the conflicts?

**Pull request closed without merging**